lib/common_models/common_models.dart

```diff
@@ -40,3 +43,27 @@ class MediaProduct {
 
   MediaProduct(this.media);
 }
+
+class MediaCategory {
+  final CT.Media media;
+
+  MediaCategory(this.media);
+}
+
+class CategoryModel {
+  final CT.Category category;
+
+  CategoryModel(this.category);
+}
+
+class RatingPaginatorModel {
```

Comment:
for paginator info, we can map it to one model because it is fixed overall application

Reply:
this not for paginator info this for Review Model